pkg/istiovalues/fips.go

```diff
@@ -0,0 +1,46 @@
+// Copyright Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package istiovalues
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/istio-ecosystem/sail-operator/pkg/helm"
+)
+
+const (
+	FipsEnableFilePath = "/proc/sys/crypto/fips_enabled"
+	TestModeFilePath   = "/tmp/fips_enabled"
+)
```

Comment:
By changing this into a `var`, tests can override it.
```suggestion
var FipsEnableFilePath = "/proc/sys/crypto/fips_enabled"
```